### ISSUE TITLE
Lintr can not be disabled with VS Code 1.21.1 on macOS 10.13.4

### ISSUE THREAD

**Issue by ukyoi** [Ikuyadeu/vscode-R#60]
- VSCode Version: 1.21.1
- VSCode-R Version: 0.5.6
- OS Version: macOS 10.13.4

Steps to Reproduce:

1. Install lintr in R terminal.
2. Open VS Code with vscode-R extension enabled.
3. set "r.lintr.enabled" to false.
4. Open an R file.

(If related)setting.json

```json
// R path for Mac OS X
"r.rterm.mac": "/usr/local/bin/R",

// Enable lintr
"r.lintr.enabled": false,

// list of linter functions
"r.lintr.linters": "default_linters",

// toggle caching of lint results
"r.lintr.cache": false,

// R executable path for lintr
"r.lintr.executable": "",

// If true, lintr exec lint_package() instead of lint()
"r.lintr.ispackage": false
```

Thanks for reading this. I found a similar issue at [here](https://github.com/Ikuyadeu/vscode-R/issues/16) while it can't be solved with upgrading to the newest version.

**Comment by adamaltmejd** [comment]
Happening for me too, with "r.lintr.enabled: false" not helping. Seems like my old ticket https://github.com/Ikuyadeu/vscode-R/issues/40 is still active, with the R session loading the `lintr` package running with --no-init-file and thus not loading the local ~/.Rprofile with libpaths() set.

**Comment by Ikuyadeu** [comment]
@ukyoi and @adamaltmejd, thank you for issue.
I think it is cause of R language server. it has lintr function.
Could you try to disable these extension?
Sorry, I can not change r language server settings now.

**Comment by ukyoi** [comment]
The linter warnings disappear if languageserver has been removed. But does it mean that the language support is not working at all?

**Comment by randy3k** [comment]
There are no settings to configure the features of the language server right now. We are working on it.

**Comment by dsteg** [comment]
@Ikuyadeu: you made R LSP Client a dependency so it cannot be disabled in VS code.

**Comment by ohmlovesampere** [comment]
How to disable lintr on Windows? Is it necessary to remove the "languagesever" package? I want to use the R LSP which needs it.

**Comment by randy3k** [comment]
@Ikuyadeu 
Diagnostics can be disabled by using the setting `diagnostics` in `package.json` of `vscode-r-lsp`. Something like

```
 "contributes": {
        "configuration": {
            "title": "R Language Server Configuration",
            "properties": {
                "diagnostics": true
.....
```

**Comment by Ikuyadeu** [comment]
Fixed by  REditorSupport/vscode-r-lsp#15
